### PR TITLE
Configure nginx server port and read timeout

### PR DESCRIPTION
**Pull request by exoszajzbuk**
This is just a small feature, which enables the user to configure the server port as well.

### CODE REVIEW

**Comment by VFedyk** [comment]
Is there anybody who can merge this to upstream?

**Comment by rhomeister** [comment]
I'll merge this if you rebase onto master and squash your commits.

**Comment by exoszajzbuk** [comment]
@rhomeister rebased and squashed. also nginx read timeout is option was added on my branch, so I included it in the PR. hope it's OK.

File: lib/generators/capistrano/unicorn_nginx/templates/_default_server_directive.erb [hunk 1]
```diff
@@ -37,7 +37,10 @@ server {
         proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
         proxy_set_header Host $http_host;
         proxy_redirect off;
-        <% if fetch(:nginx_use_ssl) -%>
+        <% if fetch(:nginx_read_timeout) %>
```
Comment (rhomeister) [inline lib/generators/capistrano/unicorn_nginx/templates/_default_server_directive.erb]: Please preserve the ERB white line trim mode.

File: lib/generators/capistrano/unicorn_nginx/templates/_default_server_directive.erb [hunk 2]
```diff
@@ -15,9 +15,9 @@ server {
     ssl on;
     ssl_certificate <%= nginx_ssl_cert_file %>;
     ssl_certificate_key <%= nginx_ssl_cert_key_file %>;
-<% else -%>
-    listen 80;
-<% end -%>
+<% else %>
```
Comment (rhomeister) [inline lib/generators/capistrano/unicorn_nginx/templates/_default_server_directive.erb]: Please preserve the ERB white line trim mode.

**Comment by exoszajzbuk** [comment]
@rhomeister fixed. do you need this commit squashed to the previous as well?

**Pull request merged**